src/tiles/selection-tile.component.ts

```diff
@@ -11,8 +11,9 @@ import { I18n } from "carbon-components-angular/i18n";
 
 @Component({
 	selector: "cds-selection-tile, ibm-selection-tile",
-	template: `
-		<input
+	template:
+		/* eslint-disable max-len */
+		`<input
```

Comment:
Any way we can ignore this for templates?

Reply:
I've tried looking for a way to achieve this but wihtout success so far.
It's probably also not the best idea to remove the length restriction for the templates. I'd suggest to for now just keep it and disable in cases it makes sense?